src/CMakeLists.txt

```diff
@@ -209,7 +120,7 @@ set(F3D_HEADER_FILES
   readers/F3DReader.h
   readers/F3DReaderFactory.h
   ${CMAKE_CURRENT_BINARY_DIR}/F3DIcon.h
-  ${CMAKE_CURRENT_BINARY_DIR}/F3DReaderInstantiator.h
+  ${CMAKE_CURRENT_BINARY_DIR}/readers/F3DReaderInstantiator.h
```

Comment:
should the readers file added from within the readers CMakeLists ?

Reply:
It's not helping, we have to instantiate them

Reply:
not sure to follow, all I mean is that since we have a CMakeLists.txt in readers directory, we could handle everything related to readers file there.

Reply:
Ok I got your point. The readers are header-only, so it's not important. Adding headers just make them visible in IDEs, but since they are CMake generated, I'd prefer to hide them anyway.